pubpub/pubpub
# update community FTUE language

Updates FTUE language.

<img width="1206" alt="Screen Shot 2020-04-21 at 09 59 10" src="https://user-images.githubusercontent.com/639110/79874775-c3c55f00-83b6-11ea-8362-6ab6e5678452.png">

_Test plan_
1. Create a new community on duqduq
1. Verify that text on homepage matches screenshot
